cli/src/cmd/new.rs

```diff
@@ -69,6 +75,13 @@ fn initialize_for_lang(name: &str) -> Result<()> {
                     fs::create_dir_all(&p)?;
                 }
             }
+            if outpath.exists() {
```

Comment:
This is racy.  Instead, the file should be opened with the `O_EXCL|O_NOFOLLOW` flags on \*nix.